atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DbAtlasDbFactory.java

```diff
@@ -42,7 +42,15 @@ public String getType() {
         return TYPE;
     }
 
-    // async initialization not implemented/propagated
+    /**
+     * Creates a ConnectionManagerAwareDbKvs.
```

Comment:
Worth noting here that, as the comment previously said, async intialization is not implemented or propagated.